### DISCUSSION TITLE
Integration in React

### DISCUSSION THREAD

**Discussion by Hamid293** [umami-software/umami#1183] [General]
Hey, thank you for creating and open-sourcing this.

I am thinking of collecting analytics for my website which is made in React. Can someone guide me on how to add umami tracker in React?
Also, I want to create a separate dashboard for each route on my site. What should be the ideal setup for each route to have its own dashboard showing its analytics?

Thank you

**Comment by MostafaaRamadan** [comment]
Did you come up with a solution on how you can integrate this with react js?

**Comment by FleetAdmiralJakob** [comment]
Hi, any updates on this?
